inst/config/link.yml

```diff
@@ -4,6 +4,14 @@ default:
     logo: "https://github.com/2DegreesInvesting/tiltWebTool/raw/main/inst/logo/logo.png"
     link: "https://www.tiltsmes.org/"
 
+  welcome:
+    text: "feedback here"
```

Comment:
If the `text` will be used as a standalone phrase or at the beginning of a sentence, then maybe we want to use "Title Case" or "Sentence case" rather than "lower case"?

Reply:
ml04: again the text in the welcome section should be after you approve my changes this one here: 

Welcome to tilt's Online Portal

Our mission is to empower Small and Medium Enterprises (SMEs) across the EU and beyond with the funding and expertise needed to thrive in a net-zero future by 2050.

tilt  (Transforming in a Low Carbon Transition) is our data analytics solution designed to provide financial institutions with critical climate data, enabling them to offer SMEs the financing and knowledge essential for green transformation. By bridging the SME data gap, tilt reduces costs for financial institutions while supporting SMEs on their sustainability journey.

Our goal is to drive investment in climate mitigation and adaptation by equipping financial institutions with the insights needed to provide SMEs with targeted financing.

Watch this video to learn more about tilt. 

What You’ll Find on This Website

tilt’s Methodology: Explore our approach through training videos. Our methodology links minimal data inputs from SMEs to climate data (e.g., climate scenarios, Life-Cycle-Assessment data), generating climate indicators that reveal a company's emission intensity and climate risks.
Research Data: Access research data available for non-commercial use.
Climate Action Guide: Discover our climate action guide, which standardizes the collection of revenue and climate action information from SMEs, improving data quality and enabling tailored sustainability loans.

These tools were developed under the CB-PASTAX program, funded by the EU. We've tested our product with five EU banks, with input from 21 banks in total. For applications beyond research, please visit www.tiltSMEs.org. Our prototype is launching with full-coverage in the Netherlands soon.

We Value Your Feedback

We strive for transparency and welcome input from financial institutions, researchers, and NGOs. Join the tilt community and share your feedback here. 


In the same logic as above I thought that the link should be embedded in this text namley at the point 'feedback here'. So looking at the above in bold: 

We strive for transparency and welcome input from financial institutions, researchers, and NGOs. Join the tilt community and share your **feedback here.**